.eslintrc.json

```diff
@@ -1,235 +1,220 @@
 {
-    "extends": [
-        "airbnb",
-        "airbnb/hooks",
-        "plugin:import/recommended",
-        "plugin:import/typescript",
-        "prettier",
-        "plugin:@next/next/recommended",
-        "plugin:i18next/recommended",
-        "plugin:storybook/recommended"
+  "extends": [
+    "airbnb",
+    "airbnb/hooks",
+    "plugin:import/recommended",
+    "plugin:import/typescript",
+    "prettier",
+    "plugin:@next/next/recommended",
+    "plugin:i18next/recommended",
+    "plugin:storybook/recommended"
+  ],
+  "parser": "@babel/eslint-parser",
```

Comment:
Can we please not change anything related to linting or prettier in this PR? Everything is already in place

Reply:
OK.

Reply:
mainly, I did not change anything.

Reply:
@osamasayed , Can you please re-confirm?